src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/AvoidNestedBlocksCheck.java

```diff
@@ -61,6 +61,55 @@
  * the allowInSwitchCase property to true and include all statements
  * of the case in the block.
  * </p>
+ * <ul>
+ * <li>
+ * Property {@code allowInSwitchCase} - Allow nested blocks if they are the
+ * only child of a switch case.
+ * Type is {@code boolean}.
+ * Default value is {@code false}.
+ * </li>
+ * </ul>
+ * <p>
+ * To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name="AvoidNestedBlocks"/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public void foo() {
+ *   int myInteger = 0;
+ *   {                      // violation
+ *     myInteger = 2;
+ *   }
+ *   System.out.println("myInteger = " + myInteger);
+ * }
+ *
+ * //example of redundant nested block in switch case
```

Comment:
no need for this comment. Please move this switch example to method above to make it compliable

Reply:
Done